### DISCUSSION TITLE
Add more QuantityCodes

### DISCUSSION THREAD

**Discussion by m4Viper** [stephanstapel/ZUGFeRD-csharp#452] [Ideas]
Hello everybody - thank you for your great work!

Is it possible to add more usual QuantityCodes to be more flexible? I think some standard and often used codes for "invoicing" are still missing.
for example: 
price per ten: TP
price per hundred: CEN
...

Or is their a easy way to add/use codes without building the code on my own?

**Comment by stephanstapel** [comment]
The quantity code list contains more than 2,000 quantity codes. That is too much for such a component.
